src/plugins/objects/bank/bank.plugin.ts

```diff
@@ -114,10 +114,12 @@ export const withdrawItem: itemInteractionActionHandler = (details) => {
     }
 
     let itemIdToAdd: number = details.itemId;
+    let stackable = details.itemDetails.stackable;
```

Comment:
For consistency's sake

`let stackable: boolean = details.itemDetails.stackable;`

Reply:
```suggestion
    let stackable: boolean = details.itemDetails.stackable;
```